app/docs/enterprise/0.31-x/getting-started/licensing.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Kong Enterprise Edition Licensing
+---
+
+# Kong Enterprise Edition Licensing
+
+## Overview
+Starting with version `0.29`, Kong Enterprise Edition enforces the presence and validity of a license file. 
```

Comment:
This is inconsistent with other mentions of the version number (of either CE or EE).

So far, I have seen:
- .30
- 0.31
- CE 0.11
- EE 0.29
- Kong `0.30`
- Kong Enterprise 0.31
- Kong Community Edition 0.12.1
- 0.12.1 (for CE, without a "CE" prefix
- `0.29` (with backticks)

Ideally we should stick to one... cc @coopr

Reply:
Agreed, My vote is for:

```
Kong Enterprise `xx.xx`
```